guides/shipments/split-shipments.html.markdown

```diff
@@ -0,0 +1,108 @@
+# Split shipments
+
+Solidus supports split shipments out of the box. This feature addresses the
+needs of complex Solidus stores who require detailed inventory management, and
+sophisticated shipping and warehouse logic. It also allows you to manage
+shipments from multiple stock locations.
+
+[`Spree::Stock::SimpleCoordinator`][simple-coordinator] contains all of the
+business logic for how stock should be packaged. If your store requires a
+specialized flow for handling split shipments, the simple coordinator should
+provide a good starting point for customizations. 
+
+## Creating proposed shipments
+
+An order's shipments are determined by
+[`Spree::Stock::SimpleCoordinator`][simple-coordinator] while the 
+`Spree::Order`'s' state is set to `delivery`. This occurs before the customer
+has completed their order at checkout.
+
+The simple coordinator deletes any existing shipments for an order and then
+creates all of the shipments currently available for this order.
+
+[simple-coordinator]: https://github.com/solidusio/solidus/blob/master/core/app/models/spree/stock/simple_coordinator.rb
+
+The simple coordinator performs a number of tasks in order to create shipment
+proposals:
+
+1. The coordinator calculates the availability of the ordered items.
+2. Inventory is allocated from available stock to the current order.
+3. It splits the order into logical packages based on stock locations and
+   inventory at those locations.
+4. Each package generates a new `Spree::Shipment` object that is associated with
+   the current order.
+5. It estimates the shipping rates for each shipment.
+ 
+After the proposed shipments have been determined, the customer can continue the
+checkout process and take the order from the `delivery` state to the `payment`
+state.
+
+## Splitters
+
+In order to split shipments, Solidus runs a series of splitters in sequence. The
+first splitter in the sequence takes the array of packages from the order,
+splits the order into packages according to its rules, then passes the packages
+on to the next splitter in the sequence.
```

Comment:
For each shipment that is thus generated, a shipping method is chosen. (this is to clarify that the splitting does not have anything to do with how the shipment is packed or shipped, instead the shipment is actually split in two separately fulfillable entities).